dev/test/specs/starter-test.js

```diff
@@ -258,7 +263,7 @@ function allSuggestions() {
   // There are two 'i' elements -- the first is 'search' and the second is
```

Comment:
Maybe don't need this comment now (I wrote it since the reason for `first-child` below wasn't readily apparent).